CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 A curated, chronologically ordered list of notable changes in [Gitpod's default workspace images](https://hub.docker.com/u/gitpod).
 
+## 2024-07-01
+
+- Bump Clang to `18`
```

Comment:
```suggestion
- Bump llvm to 18, and enable ASAN support.
```

Reply:
For consistency, I'd like to keep using CLang. [[1](https://github.com/gitpod-io/workspace-images/blob/5c71a326e421c2a09929350b095ea42cbc68a4c8/CHANGELOG.md#L80)][[2](https://hub.docker.com/r/gitpod/workspace-c)]

But, I will add ` and enable ASAN support`. Thanks!